projects/worker-azure-ai/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "azure-worker-ai",
+  "version": "0.0.1",
+  "private": true,
+  "dependencies": {
+    "@adobe/asset-compute-sdk": "^2.2.1",
```

Comment:
Update the SDK to latest?
(Check post-processing won't run and fail before updating).

Reply:
we have to update our generator :)